concepts/mev-commit-chain/chain-details.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Chain Details'
 sidebarTitle: Chain Details
 ---
 
-mev-commit chain mainnet is set to launch soon. In the meantime, join the testnet and get familiar with its functionalities from the [Testnet](https://docs.primev.xyz/developers/mev-commit-testnet) section.
+mev-commit chain mainnet is set to launch soon. In the meantime, join the testnet and get familiar with its functionalities from the [Testnet]/developers/mev-commit-testnet) section.
```

Comment:
Just passing by here :) do we need the opening parentheses? I see this pattern elsewhere too
```suggestion
mev-commit chain mainnet is set to launch soon. In the meantime, join the testnet and get familiar with its functionalities from the [Testnet](/developers/mev-commit-testnet) section.
```